### ISSUE TITLE
Please release a new version

### ISSUE THREAD

**Issue by ryandesign** [gulrak/filesystem#182]
The current version of ghc-filesystem, 1.5.14, does not work on macOS 10.12 or earlier because it uses `utimensat` which was introduced in macOS 10.13 (#168).

The PR that fixed (#167) this was merged nine months ago.

Please release a new version so users can benefit from this and other fixes you've made since the last release.

**Comment by gulrak** [comment]
Good point! I try to wrap up things for a release, this month.

**Comment by gulrak** [comment]
Hi, just want to add that I haven't forgotten to release, but I am still looking into some open issues. Have set me a deadline end of the week to release with or without those fixes. ;-)

**Comment by Shebuka** [comment]
Bump to this issue as I'm interested in a new version too 👍

**Comment by jspiers** [comment]
Yes, please release a new version. This is currently preventing me from building the `micromamba` nix package on my `x86_64-darwin` system.

**Comment by albert-github** [comment]
The latest release is, as far as I can see, 1.5.14 from March 5, 2023 so it would be very nice to have a new release.

**Comment by gulrak** [comment]
Release v1.5.16 is out now, took way too long, sorry for that!
